cli/src/trusted_base_cli/commands/pay_as_bid_proof.rs

```diff
@@ -43,8 +39,17 @@ pub struct PayAsBidProofCommand {
 impl PayAsBidProofCommand {
 	pub(crate) fn run(&self, cli: &Cli, trusted_args: &TrustedCli) {
 		println!(
-			"Result: {:?}",
-			pay_as_bid_proof(cli, trusted_args, &self.account, &self.orders_file, self.leaf_index)
+			"{:?}",
```

Comment:
It is important to only print the proof there, otherwise it is a hassle to use the output in the script.